nilearn/decoding/decoder.py

```diff
@@ -330,6 +330,18 @@ def __init__(self, estimator='svc', mask=None, cv=10, param_grid=None,
         self.n_jobs = n_jobs
         self.verbose = verbose
 
+    def _check_estimator(self):
+        if not isinstance(self.estimator, _basestring):
+            warnings.warn('Use a custom estimator at your own risk.')
+
+        elif self.estimator in list(SUPPORTED_ESTIMATORS.keys()):
+            estimator = SUPPORTED_ESTIMATORS.get(self.estimator,
+                                                 self.estimator)
+        else:
+            raise ValueError("The list of known estimator is: %s" %
```

Comment:
Saying it is a list of something is redundant and not clean grammar, as it can be seen it is a list.
Change the error message to 
```python
ValueError('Invalid estimator. Known Estimrators are:{}'.format(SUPPORTED_ESTIMATORS.keys()))
```